homeassistant/components/frontier_silicon/config_flow.py

```diff
@@ -158,13 +221,7 @@ async def async_step_device_config(
             unique_id = await afsapi.get_radio_id()
             await self.async_set_unique_id(unique_id)
```

Comment:
```suggestion
            await self.async_set_unique_id(unique_id, raise_on_progress=False)
```

We want the user flow to take precedence over any automatic discovery